src/database/queries/platformProviderResponses/selectByExternalId.sql

```diff
@@ -0,0 +1,9 @@
+SELECT
+  id AS "id",
+  external_id AS "externalId",
+  platform_provider AS "platformProvider",
+  data,
```

Comment:
Ah yes -- any preference at all on which is better (I had intended to take out the AS "id" just because I thought it's what you and jason would have suggested; I probably would value consistency)?

Reply:
I'm OK with aiming at the concise form and only doing the "AS" as needed.

Reply:
hahaha I'll take out the "AS" when it isn't needed!  ty for calling out the discrepancy.